### PR TITLE
Next videos as embedded videos on Methode articles

### PR DESCRIPTION
**Pull request by iulianp26**
Support will be provided for embedded videos within:
- videoplayer element with both ID types: UUID (from Next Video Editor) and old Brightcove (from old Methode articles)
- content element with hyperlinks to Next video

To follow tests on video cluster (only dev tests were performed)

Example of articles with: 
- videoplayer Next video: https://pub-pre-prod-uk-up.ft.com/__nativerw/methode/bb6e39a6-243e-11e7-a24c-6bbb6ec0bc98
- videoplayer Brightcove video: https://pub-pre-prod-uk-up.ft.com/__nativerw/methode/37a3327c-0fb0-11e7-a8da-f00cca55919e
- linked video: https://pub-pre-prod-uk-up.ft.com/__nativerw/methode/928c5850-243f-11e7-a24c-6bbb6ec0bc98

See documentation for Methode articles embedded videos handling:
https://sites.google.com/a/ft.com/universal-publishing/documentation/story-specs/3-mar-2017-next-video-platform-work

### CODE REVIEW

File: src/test/java/com/ft/methodearticlemapper/transformation/BaseXMLEventHandlerTest.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.ft.methodearticlemapper.transformation;
 
 import java.util.ArrayList;
+import java.util.HashMap;
```
Comment (tosan88) [inline src/test/java/com/ft/methodearticlemapper/transformation/BaseXMLEventHandlerTest.java]: Why is this added?

**Review by tosan88 — APPROVED** [review]

File: src/test/java/com/ft/methodearticlemapper/transformation/BaseXMLEventHandlerTest.java (hunk 1 above)
Comment (iulianp26) [reply src/test/java/com/ft/methodearticlemapper/transformation/BaseXMLEventHandlerTest.java]: Removed it.

**Review by tosan88 — APPROVED** [review]

**Review by ioanaciont — APPROVED** [review]

**Comment by kovacshuni** [comment]
Keep in mind, here you will need to edit when types become `Video` from `MediaResource`.

**Review by kovacshuni — APPROVED** [review]

**Review by ghost — APPROVED** [review]
